content/en/docs/concepts/storage/ephemeral-volumes.md

```diff
@@ -0,0 +1,254 @@
+aware pod scheduling](/docs/concepts/storage-capacity/) is *not*
+supported for these volumes. They are currently also not covered by
+the storage resource usage limits of a pod, because that is something
+that kubelet can only enforce for storage that it manages itself.
+
+
+Example:
+
+```yaml
+kind: Pod
+apiVersion: v1
+metadata:
+  name: my-csi-app
+spec:
+  containers:
+    - name: my-frontend
+      image: busybox
+      volumeMounts:
+      - mountPath: "/data"
+        name: my-csi-inline-vol
+      command: [ "sleep", "1000000" ]
+  volumes:
+    - name: my-csi-inline-vol
+      csi:
+        driver: inline.storage.kubernetes.io
+        volumeAttributes:
+          foo: bar
+```
+
+The `volumeAttributes` determine what volume is prepared by the
+driver. These attributes are specific to each driver and not
+standardized. See the documentation of each CSI driver for further
+instructions.
+
+Cluster administrators can control which CSI drivers can be used in a
+pod via the [Pod Security
+Policy](/docs/concepts/policy/pod-security-policy/) with the
+[`PodSecurityPolicySpec.allowedCSIDrivers` field](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.18/#podsecuritypolicyspec-v1beta1-policy).
+
+### Generic ephemeral volumes
+
+{{< feature-state for_k8s_version="v1.19" state="alpha" >}}
+
+This feature requires the GenericEphemeralVolume feature gate to be
+enabled. Because this is an alpha feature, it is disabled by default.
+
+Generic ephemeral volumes are similar to `emptyDir` volumes, just more
+flexible:
+- Storage can be local or network-attached.
+- Volumes can have a fixed size that pods are not able to exceed.
+- Volumes may have some initial data, depending on the driver and
+  parameters.
+- All of the normal volume operations
+  ([snapshotting](/docs/concepts/storage/volume-snapshots/),
+  [cloning](/docs/concepts/storage/volume-pvc-datasource/),
+  [resizing](/docs/concepts/storage/persistent-volumes/#expanding-persistent-volumes-claims),
+  [storage capacity tracking](/docs/concepts/storage-capacity/), etc.)
+  are supported.
+
+Example:
+
+```yaml
+kind: Pod
+apiVersion: v1
+metadata:
+  name: my-app
+spec:
+  containers:
+    - name: my-frontend
+      image: busybox
+      volumeMounts:
+      - mountPath: "/scratch"
+        name: scratch-volume
+      command: [ "sleep", "1000000" ]
+  volumes:
+    - name: scratch-volume
+      ephemeral:
+        volumeClaimTemplate:
+          metadata:
+            labels:
+              type: my-frontend-volume
+          spec:
+            accessModes: [ "ReadWriteOnce" ]
+            storageClassName: "scratch-storage-class"
+            resources:
+              requests:
+                storage: 1Gi
+```
+
+### Lifecycle and PersistentVolumeClaim
+
+The key design idea is that the [parameters for a
+PersistentVolumeClaim](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.19/#ephemeralvolumesource-v1alpha1-core)
```

Comment:
Yes. That "PersistentVolumeClaim" has "persistent" in it is now mostly historic: "VolumeClaim" would have been a more appropriate name for the API. We can't fix the API, but in prose we can just refer to it as "volume claim".

I changed the link text but then kept `PersistentVolumeClaim object` in the rest of the sentence because technically that's what it is.